packages/jest/src/jest-configuration-builder.ts

```diff
@@ -24,12 +54,7 @@ export class JestConfigurationBuilder {
   buildConfiguration(projectRoot: Path, workspaceRoot: Path, configPath: string = 'jest.config.js'): any {
```

Comment:
No need in default argument anymore, since you're using buildConfiguration function